docs/configuration.md

```diff
@@ -246,7 +246,7 @@ the standard [`nuget push`][nuget-push] command.
 
 [NuGet]: https://www.nuget.org
 [nuget-push]: https://docs.microsoft.com/fr-fr/nuget/nuget-org/publish-a-package
-[.NET documentation]: https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagelicenseurl
+[.NET documentation]: https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packageiconurl
```

Comment:
This doesn't seem right. I think the #link was extraneous to begin with. Remove it?

[.NET documentation]: https://docs.microsoft.com/en-us/dotnet/core/tools/csproj

Reply:
Nah the link is in fact used a couple of lines above to point people at where Microsoft tells you what icons they suggest you use. It's not absolutely necessary, but it's actually kinda useful (as at least this can be expected to remain current on what Microsoft recommends)